pkg/build/builder/sti.go

```diff
@@ -33,7 +33,7 @@ func NewSTIBuilder(client DockerClient, dockerSocket string, authCfg docker.Auth
 func (s *STIBuilder) Build() error {
 	tag := s.build.Parameters.Output.DockerImageReference
 	request := &stiapi.Request{
-		BaseImage:    s.build.Parameters.Strategy.STIStrategy.Image,
+		BaseImage:    s.build.Parameters.Strategy.STIStrategy.From.Name,
```

Comment:
@smarterclayton keep me honest, i'm not sure how that plays in with access control to docker images...